<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecRulesPage</class>
 <widget class="QWidget" name="SecRulesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_14">
     <property name="title">
      <string>安全策略</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>允许规则</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="1">
           <widget class="QComboBox" name="domainTypeBox"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_24">
            <property name="maximumSize">
             <size>
              <width>66</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>进程类型</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_26">
            <property name="maximumSize">
             <size>
              <width>66</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>class类型</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_25">
            <property name="maximumSize">
             <size>
              <width>66</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>文件类型</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QComboBox" name="classBox"/>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="fileTypeBox"/>
          </item>
          <item row="0" column="6">
           <widget class="QPushButton" name="findButton">
            <property name="text">
             <string>查找</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="7">
           <widget class="QTableWidget" name="ruleTableWidget">
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>进程类型</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>文件类型</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Class类型</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>权限信息</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>文件/进程类型转换</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="0">
           <widget class="QLabel" name="label_28">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>源进程类型</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="sourceTypeBox"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_27">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>目标类型</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="targetTypeBox"/>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_29">
            <property name="maximumSize">
             <size>
              <width>63</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>class类型</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QComboBox" name="classTypeBox"/>
          </item>
          <item row="0" column="6">
           <widget class="QPushButton" name="findButton2">
            <property name="text">
             <string>查找</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="7">
           <widget class="QTableWidget" name="f_p_tableWidget">
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>源进程类型</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>目标类型</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Class类型</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>默认类型</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
